hangman/hangman.py

```diff
@@ -1,27 +1,45 @@
 import random
-my_list = ["I", "eat", "chicken", "and", "turkey"]
-random_word = str(random.choice(my_list).upper())
-print(random_word)
+def get_target_word():
+    my_list = ["I", "eat", "chicken", "and", "turkey"]
+    random_word = str(random.choice(my_list).upper())
+    print(random_word)
+    return random_word
+random_word = get_target_word()
+# To use the variable outside the function
 
-dic_list = {}
-for letters in random_word:
-    dic_list[letters] = False
-print(dic_list)
+def convert_to_dictionary():
+    # this function helps determine which letter 
+    # from the target word is guessed correctly. 
```

Comment:
What does the function do? You're not saying anything here

Reply:
The name is self explanatory

Reply:
The name is self explanatory